File: resolucoes/java/lista01/src/main/java/org/example/Ex007.java
/*7.
 Crie um algoritmo que leia um número real e mostre na tela o seu dobro e a 
sua terça parte. ex:
Digite um número: 3.5
O dobro de 3.5 é 7.0
A terça parte de 3.5 é 1.16666
 */

package org.example;

import java.util.Scanner;

public class Ex007 {
    public static void main(String[] args) {
        Ex007.main();

    }

    public static void main() {

        Scanner sc = new Scanner(System.in);
        System.out.print("Digite um número: ");
        Float num = sc.nextFloat();
        System.out.printf("O dobro de %s é %s\n", num, num * 2);
        System.out.printf("A terça parte de %s é %s", num, num / 3);
        sc.close();

    }

}

File: resolucoes/php/lista01/ex007.php
<?php
/*
Crie um algoritmo que leia um número real e mostre na tela o seu dobro e a sua terça parte. ex:
Digite um número: 3.5
O dobro de 3.5 é 7.0
A terça parte de 3.5 é 1.16666
*/

$inputValue = readline('Digite um número: ');

if (preg_match('/^[+-]?\d*\.?\d+$/', $inputValue)) {
  $number = floatval($inputValue);
  $dobro = 2 * $number;
  $terca = round(($number/3),5);

  print "O dobro de $number é $dobro \n";
  print "A terça parte de $number é $terca.\n";
} else {
  echo "A entrada deve conter apenas números.";
}